src/main/resources/db/changelog/changes/04-create-user_role-table.yaml

```diff
@@ -0,0 +1,18 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-user_role-table
+      author: tania
+      changes:
+        - createTable:
+            tableName: user_role
+            columns:
+              - column:
+                  name: user_id
+                  type: bigint
+                  constraints:
+                    nullable: false
+              - column:
+                  name: role_id
+                  type: bigint
+                  constraints:
+                    nullable: false
```

Comment:
add constraints to this table:

PK (user_id, role_id)
FK (user_id)
FK (role_id)

Reply:
Yes, add constraints please